kubernetes-client/src/main/java/io/fabric8/kubernetes/client/dsl/Gettable.java

```diff
@@ -23,4 +25,11 @@
    */
   T get();
 
+
+  /**
+   * @return the item or throws Exception if the item doesn't exist.
+   * @throws io.fabric8.kubernetes.client.KubernetesClientException if an error occurs
+   */
+  T getRequired() throws ResourceNotFoundException;
```

Comment:
TBH, I don't like mustGet, I also don't like getRequired(), getMandatory() or even require().

I was tempted to make it part of the DSL as an additional method like: .failingIfNotExists().get(); But that also seems like an overkill. So not sure ....